internal/resources/webform/webform-sample.css

```diff
@@ -411,6 +411,11 @@ table#grpc-request-metadata-form {
     padding: 0;
 }
 
+textarea#grpc-response-textarea {
+    font-family: Monoca, monospace;
```

Comment:
Let's please be consistent and just use the same font/formatting as other text areas.

Reply:
updated!